test/endToEndTests.ts

```diff
@@ -43,7 +43,7 @@ describe('satcheljs', () => {
         let arg1Value;
         let arg2Value;
 
-        let testMutatorAction = mutatorAction('testMutatorAction', function testMutatorAction(
+        let testMutatorAction = mutatorAction<void>('testMutatorAction', function testMutatorAction(
```

Comment:
><void> [](start = 45, length = 6)

Ugh.  It makes you put this in?  Can we default it to `void` so this isn't necessary?